server/backend/testspg/router/test_router_application.py

```diff
@@ -0,0 +1,21 @@
+import logging
+import starlette.testclient
+from api.app.main import apiPrefix
+import tests.tests.jwt_utils as jwt_utils
+
+LOGGER = logging.getLogger(__name__)
+endPoint = f"{apiPrefix}/fam_applications"
+
+
+# def test_get_applications(
+#     test_client_fixture: starlette.testclient.TestClient,
```

Comment:
comment out this test, as it complains with the sqlite tests, but when run it locally without sqlite tests, it works fine